### PR TITLE
Service changes

### PR DESCRIPTION
**Pull request by Hunttark**
feedback thread: https://forums.aurorastation.org/topic/18084-feedback-thread-for-service-remap

---

* The bar lounges have been removed in favor of a central dining hall.
* The Bar has been moved to the starboard side, with additional bar seats and distance from the eating area to prevent convo-clashing.
* The kitchen has remained mostly the same, besides gaining 4 tabletop surfaces.
* Shutters were provided to the exterior windows that can be controlled by either the bar, dining hall or kitchen respectively depending on where the (carp) are at the time.
* Pun Pun has a little bedroom with bananas now.

Reference old PR that closed when I sync'd branches: https://github.com/Aurorastation/Aurora.3/pull/15961

## screenshots
<details>
<summary>🖼️ click to view</summary>

e7672bac5ce891fae499cc70fb1f8b5c

1dbf5638b6f05f8ad4027d4a6e306c64
</details>

_(edited by SleepyGemmy @ 2023-03-07: added feedback thread link. added a screenshots section.)_

### CODE REVIEW

**Comment by Hunttark** [comment]
!review

**Comment by Hunttark** [comment]
@SleepyGemmy @RyverStyx @alsoandanswer

File: html/changelogs/Huntime-Servicechanges.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Remapped service area."
```
Comment (SleepyGemmy) [inline html/changelogs/Huntime-Servicechanges.yml:41]:
```suggestion
  - maptweak: "Remapped the service area."
```

**Review by SleepyGemmy — APPROVED** [review]

File: html/changelogs/Huntime-Servicechanges.yml (hunk 1 above)
Comment (Hunttark) [reply html/changelogs/Huntime-Servicechanges.yml]: Updated.

**Review by alsoandanswer — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
- stray button in the kitchen.
image
- the lounge viewing shutters button and shutters IDs do not match.
- a security camera in hydroponics will be hard to repair.
image
- the bar viewing shutters button isn't public.
- the audioconsole near the dining hall entrance isn't secured (anchored?).

**Review by SleepyGemmy — APPROVED** [review]

**Review by RyverStyx — APPROVED** [review]

**Pull request merged**